README.md

```diff
@@ -16,33 +15,31 @@ It can be considered as a specific implementation of [local persistent volumes](
 Our supported platform are:
 
 - Kubernetes: 1.27, 1.26, 1.25
-- Node OS: Linux with LVM2 (*1)
+- Node OS: Linux with LVM2 (\*1)
+- CPU Architecture: x86 (\*2), arm64 (\*3), ppc64le (\*3)
 - Filesystems: ext4, xfs, btrfs(experimental)
 - lvm version 2.02.163 or later (adds JSON output support)
 
-*1 The host's Linux Kernel must be v4.9 or later which supports `rmapbt` and `reflink`, if you use xfs filesystem with an official docker image.
+\*1 The host's Linux Kernel must be v4.9 or later which supports `rmapbt` and `reflink`, if you use xfs filesystem with an official docker image.  
+\*2 Tier1 support. The official docker images are provided and all functionalities are tested by CI.  
+\*3 Tier2 support. The official docker images are provided, but no tests run by CI.  
 
 Docker images are available on [ghcr.io](https://github.com/orgs/topolvm/packages).  
 
-Getting started
----------------
+## Getting Started
 
-A demonstration of TopoLVM running on [kind (Kubernetes IN Docker)][kind] is available at [example](example/) directory.
+If you want to try TopoLVM on your local machine, see [example](example/) directory which provides scripts to run TopoLVM on [kind (Kubernetes IN Docker)][kind].
 
-For production deployments, see [deploy/README.md](./deploy/README.md).
+To use TopoLVM on your real kubernetes cluster, read [documents](docs/README.md).
 
-User manual is at [docs/user-manual.md](docs/user-manual.md).
+_Deprecated: If you want to use TopoLVM on [Rancher/RKE](https://rancher.com/docs/rke/latest/en/), see [docs/deprecated/rancher/README.md](docs/deprecated/rancher/README.md.md).
```

Comment:
```suggestion
Deprecated: If you want to use TopoLVM on [Rancher/RKE](https://rancher.com/docs/rke/latest/en/), see [docs/deprecated/rancher/README.md](docs/deprecated/rancher/README.md.md).
```

Reply:
I noticed that enclosing texts with underscores makes them to be italics.
Anyway, the normal is fine, I'll update it.